GPSTest/build.gradle

```diff
@@ -106,9 +106,9 @@ task askForPasswords << {
                     label(text: "Key passphrase:")
                     textField id: "keyText", input = passwordField()
                     button(defaultButton: true, text: 'OK', actionPerformed: {
-                        storePw = storeText.text;
-                        keyPw = keyText.text;
-                        dispose();
+                        storePw = storeText.text
```

Comment:
Could you please revert this modification to the signing script?  It shouldn't be in the PR unless it's been tested, so let me know if you tried to sign a build release to test that this works.

Reply:
quite sure it worked

Android Studio 3.2 Beta 1 (which i should've been using 3.1.3 tbh) synced and built the app perfectly with these changes i made to the code, and was able to produce a signed release apk

Reply:
ill post an apk that Android Studio built and signed after i made these commits (had to compress them into a zip file because GitHub doesnt support apks (its the only apk btw)

[debug.zip](https://github.com/barbeau/gpstest/files/2147384/debug.zip)

Reply:
Release builds are done at command line - see https://github.com/barbeau/gpstest/blob/master/README.md#release-builds

Reply:
i tried the google debug apk i posted on my Samsung Galaxy S4 (5.0.1) and it worked fine as far as i could tell

Reply:
ill try building the release apks

Reply:
aight they didnt work i undid them